archivesunleashed/auk-notebooks
# Include functions for testing existence of derivatives. #10

This is a response to #10 that includes the following:
- Functions for accessing derivatives for graphml, gexf, and filtered text. (The latter has a placeholder only TextIO object until I figure out a logical way to process the zipped data).
- A function to test the existence of derivatives.
- Bool Vars TEXT, DOMAINS & NETWORK for easy checking.
- `thing if TEXT else []` syntax to produce empty graphs instead of an error when no derivative exists.

To test:

I've been changing the filenames from `4867-fullurls.txt` to `NOT4867-fullurls.txt` and then running the windows to see how it works.  Then you just remove the NOT to restore the original.
